### PR TITLE
Correct language in API code examples

### LINKED ISSUE #2162: Text code blocks being marked incorrectly as Python
We're using `python` as the language for code blocks that are set to `text` in Sphinx.

For example:

https://github.com/Qiskit/documentation/blob/3e46446e86830b4a5104561f73e3dce5b2ad7951/docs/api/qiskit/dev/circuit_library.mdx?plain=1#L791-L799

But Sphinx sets it to `text`:

https://github.com/Qiskit/qiskit/blob/35c03913e4fae622f3ea0e68ba22f919dca103f9/qiskit/circuit/library/templates/nct/template_nct_7b_1.py#L22-L30

### PR DESCRIPTION
**Pull request by frankharkins**
Fixes #2162 and regens the current API docs to demonstrate.

### CODE REVIEW

File: docs/api/qiskit-ibm-runtime/0.18/qiskit_ibm_runtime.transpiler.passes.scheduling.PadDynamicalDecoupling.mdx [hunk 1]
```diff
@@ -13,7 +13,7 @@ python_api_name: qiskit_ibm_runtime.transpiler.passes.scheduling.PadDynamicalDec
 
   This pass works on a scheduled, physical circuit. It scans the circuit for idle periods of time (i.e. those containing delay instructions) and inserts a DD sequence of gates in those spots. These gates amount to the identity, so do not alter the logical action of the circuit, but have the effect of mitigating decoherence in those idle periods. As a special case, the pass allows a length-1 sequence (e.g. \[XGate()]). In this case the DD insertion happens only when the gate inverse can be absorbed into a neighboring gate in the circuit (so we would still be replacing Delay with something that is equivalent to the identity). This can be used, for instance, as a Hahn echo. This pass ensures that the inserted sequence preserves the circuit exactly (including global phase).
 
-  ```python
+  ```ipython3
```
Comment (Eric-Arellano) [inline docs/api/qiskit-ibm-runtime/0.18/qiskit_ibm_runtime.transpiler.passes.scheduling.PadDynamicalDecoupling.mdx:16]: This is weird. Maybe normalize it to `python`?

**Comment by Eric-Arellano** [comment]
GitHub is having issues loading things. Maybe interactive rebase and force push? It will pull in the addons PR I recently landed so that the diff in those individual commits is smaller.

**Comment by frankharkins** [comment]
Hold off review: I'm regenerating these again and will force-push but it's gonna take a while

File: docs/api/qiskit/0.19/circuit.mdx [hunk 2]
```diff
@@ -111,7 +111,7 @@ From the plot, it is easy to see that this circuit has 12 qubits, and a collecti
 qc.width()
 ```
 
-```python
+```none
```
Comment (frankharkins) [inline docs/api/qiskit/0.19/circuit.mdx:114]: I thought this was a bug, but it's explicitly defined in the artifact

```html
<div class="output text_plain highlight-none notranslate"><div class="highlight"><pre><span></span>12
```

Comment (frankharkins) [reply docs/api/qiskit/0.19/circuit.mdx:114]: I'm gonna normalize this to nothing

Comment (Eric-Arellano) [reply docs/api/qiskit/0.19/circuit.mdx:114]: Sg. Bump on normalizing ipython3 to python (comment above)

**Review by Eric-Arellano — APPROVED** [review]
Thank you!

**Pull request merged**